cpp_linter_hooks/clang_tidy.py

```
@@ -11,9 +11,8 @@
 
 def run_clang_tidy(args=None) -> Tuple[int, str]:
     hook_args, other_args = parser.parse_known_args(args)
-    tool_name = ensure_installed("clang-tidy", hook_args.version)
-    command = [tool_name]
-    command.extend(other_args)
+    ensure_installed("clang-tidy", hook_args.version)
+    command = ["clang-tidy"] + other_args
```

Ruff (0.12.2)

Consider `["clang-tidy", *other_args]` instead of concatenation

Replace with `["clang-tidy", *other_args]`

(RUF005)